content/docs/200-inputs/600-multi-select.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Multi-select
+excerpt: Select from a large list of multi-select options.
+---
+
+Add a _multi-select_ input to your app to allow users select from a large list of multi-select options.
+
+## Options
+
+Add options to your _multi-select_ input and set a default option. The input ID in the URL template for your _multi-select_ input is replaced with the selected option’s values.
```

Comment:
I think we probably need to talk about how these options are passed in via the query string and how to decode them here too.

Reply:
Good call! See below.

https://github.com/mirainc/developer-docs/pull/42/files#diff-1b6bce6314f8c4e245fb5677670f12e1991bba222d644d0b1ae9f3a867000c0dR12-R27